Packs/CommonScripts/Scripts/GetEnabledInstances/GetEnabledInstances.py

```diff
@@ -0,0 +1,32 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_enabled_instances():
+    enabled_instances = []
+    instances = demisto.getModules()
+    for instance_name, data in instances.items():
+        if data.get('state') == 'active':
+            enabled_instances.append(instance_name)
+
+    readable_output = [
+        {
+            'Instance Name': instance,
+            'Brand': instances[instance].get('brand')
+        } for instance in enabled_instances
+    ]
```

Comment:
One loop instead of two - abit more efficient.
```suggestion
    readable_output = []
    for instance_name, data in instances.items():
        if data.get('state') == 'active':
            enabled_instances.append(instance_name)
            readable_output.append({
                'Instance Name': instance_name,
                'Brand': data.get('brand')
            })
```